app/views/admin/dashboard/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<html>
+
+<head>
```

Comment:
I think normally this would go in views/layouts?

Reply:
Updated this to be in views/admin/layouts